docker-compose-workflows.yml

```diff
@@ -97,15 +100,31 @@ services:
       - temporal
     environment:
       - SERVICE_TYPE=WORKFLOW_TEMPORAL_WORKER
-      - NODE_OPTIONS=--max_old_space_size=1024
       - DISABLE_DATABASE_MIGRATIONS=true
       - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_HOST=temporal
       - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_PORT=7233
       - WORKFLOW_BACKEND_HOST=http://api:3000
+      - CODE_EXECUTOR_INGRESS_DOMAIN=http://code-executor:3004
     networks:
       - backend-network
       - db-connector-network
       - workflows-network
+      - code-executor-network
+    restart: on-failure
+
+  code-executor:
+    build:
+      context: ./
+      dockerfile: DockerfileCodeExecutor
```

Comment:
ultra nit: I believe naming convention would have this be `CodeExecutor.Dockerfile`